### PR TITLE
Work around github by just querying drone itself, add full ci support [full ci]

### PR DESCRIPTION
**Pull request by mhagen-vmware**
Finally (I think) came up with at least a work around solution until we upgrade to drone .5

### CODE REVIEW

File: tests/integration-test.sh [hunk 1]
```diff
@@ -20,12 +20,10 @@ gsutil version -l
 
 dpkg -l > package.list
 
-echo $DRONE_BUILD_URL
-echo $DRONE_BUILD_DIR
-echo $DRONE_PULL_REQUEST
-
 if [ $DRONE_BRANCH = "master" ] && [ $DRONE_REPO = "vmware/vic" ]; then
     pybot --removekeywords TAG:secret tests/test-cases
+elif grep -q "\[full ci\]" <(drone build info vmware/vic $DRONE_BUILD_NUMBER); then
```
Comment (hickeng) [inline tests/integration-test.sh]: let's be consistent with `[full-ci]` given it's `[skip-ci]`

Comment (mhagen-vmware) [inline tests/integration-test.sh]: It is actually not skip-ci... it is skip ci.  Loc continues to use skip-ci and it doesn't work, per:
https://ci.vmware.run/vmware/vic/4344

**Comment by hickeng** [comment]
Okay, LGTM then

**Comment by kjplatz** [comment]
:+1:

**Pull request merged**